mission/config/subconfigs/vehicle_respawn_info.hpp

```diff
@@ -933,4 +978,63 @@ class spawn_point_types {
 			};
 		};
 	};
+
+	class boat_light {
+		name = "Light Boats";
+		RESPAWN_MEDIUM;
+
+		class categories {
+			class light_transport_m2 {
+				name = "STAB M2";
+				icon = VEHICLE_ICON_BOAT;
+				include[] = { { "light", "boat", "m2" } };
+			};
+			class light_transport_m60 {
+				name = "STAB M60";
+				icon = VEHICLE_ICON_BOAT;
+				include[] = { { "light", "boat", "m60" } };
+			};
+			class light_transport_mk18 {
+				name = "STAB MK18";
+				icon = VEHICLE_ICON_BOAT;
+				include[] = { { "light", "boat", "mk18" } };
+			};
+		};
+	};
+
+	class boat_large {
+		name = "Large Boats";
```

Comment:
These are all nasty boats, so might make a bit more sense to have it like this
```suggestion
	class boat_nasty {
		name = "Boats - Nasty";
```